src/homework/07/index.tsx

```diff
@@ -41,7 +44,7 @@ function human(name, age) {
 
 // Q5
 
-function handleQuantity(quantity) {
+function handleQuantity(quantity : number) :number | undefined {
```

Comment:
```suggestion
function handleQuantity(quantity : number) :number | void {
```
함수가 아무것도 리턴하지 않을 때는 undefined 대신 void 를 리턴한다고 적습니다.